modules/custom/az_block_types/az_custom_menu_block/az_custom_menu_block.module

```diff
@@ -31,3 +31,30 @@ function az_custom_menu_block_form_block_content_az_custom_menu_block_form_alter
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function az_custom_menu_block_preprocess_block(&$variables) {
+  if ($variables['base_plugin_id'] === 'menu_block') {
+    if (isset($variables['configuration']['az_custom_menu_block']) &&
+        $variables['configuration']['az_custom_menu_block'] === 'az_custom_menu_block'
+    ) {
+      $variables['title_attributes']['class'][] = 'h4';
+      $variables['title_attributes']['class'][] = 'mt-0';
```

Comment:
Is there any reason not to combine these nested if statements?

```suggestion
  if ($variables['base_plugin_id'] === 'menu_block' && 
    isset($variables['configuration']['az_custom_menu_block']) &&
    $variables['configuration']['az_custom_menu_block'] === 'az_custom_menu_block'
  ) {
    $variables['title_attributes']['class'][] = 'h4';
    $variables['title_attributes']['class'][] = 'mt-0';
```

Reply:
We could probably remove the base_plugin_id check altogether.